### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by DariaSlayer**
- [DEMO LINK](https://DariaSlayer.github.io/layout_calendar/)
- [TEST REPORT LINK](https://DariaSlayer.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+  .calendar--month-length--#{$length} :nth-child(n + #{$length + 1}) {
+    display: none;
+  }
+}
+
+@each $day, $indent in $week {
+  .calendar--start-day--#{$day} :first-child {
+    margin-left: $indent;
+  }
 }
```
Comment (denys-danyliuk) [inline src/styles/main.scss]: move them to the calendar file

File: src/styles/variables.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+$cell-size: 100px;
+$cell-gap: 1px;
+$calendar-padding: 10px;
+
+$week:
+  mon 0,
```
Comment (denys-danyliuk) [inline src/styles/variables.scss]:
```suggestion
  "mon" 0,
```

File: src/styles/variables.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  tue calc(#{$cell-size} + #{$cell-gap}),
+  wed calc((#{$cell-size} + #{$cell-gap}) * 2),
```
Comment (denys-danyliuk) [inline src/styles/variables.scss]:
```suggestion
  "wed" ($cell-size + $cell-gap) * 2,
```

File: src/styles/variables.scss (hunk 2 above)
Comment (denys-danyliuk) [reply src/styles/variables.scss]: add quotation marks around this values

File: src/styles/calendar.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+  &__day {
+    box-sizing: border-box;
+    border: 1px solid #000;
+    width: $cell-size;
+    height: $cell-size;
+    list-style: none;
+    cursor: pointer;
+    background-color: #eee;
+    transition: 500ms;
```
Comment (denys-danyliuk) [inline src/styles/calendar.scss]: 8'th point in checklist

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]
Good job, small fixes left. Your calendar should be operational, I believe after you'll add quotation marks around day values everything will work as expected. For example code below should move 1'st cell to the friday position and so on
```
calendar--start-day--fri

**Comment by DariaSlayer** [comment]
> Good job, small fixes left. Your calendar should be operational, I believe after you'll add quotation marks around day values everything will work as expected. For example code below should move 1'st cell to the friday position and so on
> 
> ```
> calendar--start-day--fri
> ```

sure, but when I tested without quotations it worked as well

**Review by andriy-shymkiv — APPROVED** [review]

**Pull request closed without merging**